setup.py

```diff
@@ -42,6 +42,7 @@
     "django-extensions>=3.0.3",
     "dateparser>=1.0.0",
     "youtube-dl>=2021.04.17",
+    "yt-dlp>=2021.4.11",
```

Comment:
I arbitrarily picked this version number to be the first datestamp ahead of the youtube-dl version.

Reply:
you can bump this to the latest version whenever you touch these files (as long as it works), no need to match older versions of chrome/youtubledl/ffmpeg/git/wget, only django and a couple other edge cases are the ones that cant be bumped